front/app/containers/App/index.tsx

```diff
@@ -78,12 +78,25 @@ import { Locale } from 'typings';
 import { removeLocale } from 'utils/cl-router/updateLocationDescriptor';
 import openSignUpInModalIfNecessary from './openSignUpInModalIfNecessary';
 
-const Container = styled.div`
+const Container = styled.div<{
+  fullScreenModal?: boolean;
+  isModalOpen?: boolean;
+}>`
   display: flex;
   flex-direction: column;
   align-items: stretch;
   position: relative;
   background: #fff;
+
+  // if the platform has fullscreen login modal enabled and the modal is open,
+  // disable scrolling on the page behind the modal
+  ${(props: any) =>
+    props.fullScreenModal &&
```

Comment:
Maybe this component doesn't need to know all the details about the modal, and we could just pass the `disableScroll` (or any other name) parameter? 

And then calling it like `<Container disableScroll={fullScreenModal && isModalOpen}>`.

But not 100% sure about it.